docs/pages/card.md

```diff
@@ -178,3 +178,78 @@ You can either set the width of cards with custom css or add them into the Found
   </div>
 </div>
 ```
+
+## Card columns
+
+Cards can be organized into <a href="http://masonry.desandro.com/" target="_blank">Masonry-like</a> columns with just CSS by wrapping them in .card-columns. Cards are built with <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns/Using_multi-column_layouts" target="_blank">CSS column</a> properties instead of flexbox for easier alignment. Cards are ordered from top to bottom and left to right.
```

Comment:
How do these interact with the grid? Can they be used with the grid? interact with it in some way?